Add travel/venue information

mrmundt opened the pull request:
## Description

This PR adds the initial information about the venue and nearby hotels, plus other travel information (e.g, driving/parking instructions).

![Preview of Travel Information Page](https://user-images.githubusercontent.com/55767766/229569709-878f3505-c08f-46a8-b7b6-4adb85d9f4a7.png)

## Motivation and Context
We want to inform people of details about the venue.

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2023-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @mrmundt @cabejackson @manning-ncsa @jmelot @jbteves @jsubida

manning-ncsa commented:
Looks good from a quick build in terms of formatting (I have not reviewed the content). In the process I noticed that if you click Attend in the nav bar and/or Attend > Register you see "We’ll post information when it becomes available.". Perhaps that message should be removed from the Attend page and replaced with a link to the Register and Travel pages.

sci-tab commented on pages/attend/travel.md:
```diff
@@ -4,7 +4,78 @@ title: Travel
+## Venue Information
+
+- **Dates**: October 16-18, 2023
+- **Location**: University of Illinois in Chicago – [Student Center East](https://meetings.uic.edu/facilities/student-center-east/) – 750 S. Halsted, Chicago, IL 60607
```
Is this a good place to add this information? https://usrse.slack.com/archives/C04E9M6NW7P/p1679922252568479 

Mostly interested in ADA and this link https://oae.uic.edu/resources/gender-inclusive-resources/

mrmundt replied:
I'd say that's entirely up to you. Where would you like that information?

sci-tab replied:
I think it might be! Let me confirm with the other co-chairs

sci-tab replied:
Yes, that's perfect @mrmundt !! I just confirmed this is a good place to include that. Thanks for taking my question and implementing it

mrmundt commented:
> Looks good from a quick build in terms of formatting (I have not reviewed the content). In the process I noticed that if you click Attend in the nav bar and/or Attend > Register you see "We’ll post information when it becomes available.". Perhaps that message should be removed from the Attend page and replaced with a link to the Register and Travel pages.

Good call! Fixed!

manning-ncsa commented on pages/attend/attend.md:
```diff
@@ -7,4 +7,11 @@ permalink: attend/
-We'll post information when it becomes available.
+## Register
+
+See the [Registration page]({{ site.baseurl }}/attend/travel) for more details.
```
Oops, looks like copypasta. You probably want /attend/register here.

mrmundt replied:
Siiigggghhh. Copy-pasta failed me again. Fixed.

manning-ncsa replied:
Fixed

manning-ncsa commented:
Looks fine to me. I'm curious what the `google.ca` addresses for the maps are about. How is `google.ca` different from `google.com`?

mrmundt commented:
> Looks fine to me. I'm curious what the `google.ca` addresses for the maps are about. How is `google.ca` different from `google.com`?

Not a clue!

mrmundt commented:
> Looks fine to me. I'm curious what the `google.ca` addresses for the maps are about. How is `google.ca` different from `google.com`?

@manning-ncsa  - Jk, I looked it up - that's the Canadian-specific domain that shows more Canadian-specific results. I changed it to the generic `google.com` URL. (Why it was the Canadian one - no idea. It was a copy-pasta from the original document as well.)

jbteves reviewed (approved):
One minor comment but looks good to me.

mrmundt commented on pages/attend/travel.md:
```diff
@@ -4,7 +4,87 @@ title: Travel
+
+## Hotel Accommodation
+
+We’re reserving hotel room blocks at several nearby hotels. Please check back
```
Unfortunately, I have no idea.
